src/aks-preview/HISTORY.rst

```diff
@@ -14,6 +14,7 @@ Pending
 * Vendor new SDK and bump API version to 2023-10-02-preview.
 * Update the enum for `--os-sku` in command `az aks nodepool update` to only accept the expected Ubuntu and AzureLinux OSSKUs.
 * Update description `az aks update` and remove description about outbound ip limit.
+* Fix the response format for `az aks mesh get-revisions` and `az aks mesh get-upgrades`. 
```

Comment:
Please rebase from main, the changes under your pending section have been released in version 0.5.172